modules/hdf5_handler/HDF5RequestHandler.cc

```diff
@@ -623,6 +623,13 @@ temp_table->print(cerr);
 
         bdas->clear_container() ;
     }
+    catch(BESSyntaxUserError & e) {
```

Comment:
I think while this gets fixed (the order of these exceptions in the catch clauses) we should add `const` to the caught exception object declaration. So, "BESSyntaxError &e" becomes "const BESSyntaxUserError &e". The same change should be applied to the BESError &e declarations, too.

Reply:
Done.